active_directory_201.rst

```diff
@@ -20,10 +20,81 @@ DHCP
 Global Catalog
 --------------
 
+The Global Catalog (GC) holds a subset of attributes for all objects in a multi-domain forest.
+It can be used to search all objects in the forest.
+For example, when searching for the a specific user, the GC can be searched to find users not only in the current domain but also in other domains in the forest.
+The attributes included in the GC's subset of object attributes are those required by the schema and those most commonly used in searches.
+Attributes can be added and removed from the Global Catalog using the Active Directory Schema snap-in.
 
-Advanced Active Directory Maintenance
-=====================================
+When Active Directory Domain Services is installed,  the GC is added to the first domain controller in the forest.
+It can be added to additional domain controllers.
+In order to avoid traversing a WAN to gather GC information, it is recommended that each Active Directory site have at least one Global Catalog server.
+
+Universal groups can contain objects from any domain in the forest.
+The GC holds the membership for all universal groups.
+As part of the authorization process, the authenticating DC retrieves the security identifier for all the member's security groups, which includes universal groups.
+Therefore, if the environment has more than one domain, users need to be able to access a global catalog server for authentication in order to find objects from the universal groups in other domains.
```

Comment:
This could be followed with a reinforcing "and thereby not having to traverse the WAN..." or something similar to conclude the reasoning for multiple GCs.